yarn.lock

```diff
@@ -1039,10 +1039,14 @@ tslint@latest:
     tsutils "^1.1.0"
     update-notifier "^2.0.0"
 
-tsutils@^1.1.0, tsutils@^1.2.2:
+tsutils@^1.1.0:
   version "1.2.2"
   resolved "https://registry.yarnpkg.com/tsutils/-/tsutils-1.2.2.tgz#7e165c601367b9f89200b97ff47d9e38d1a6e4c8"
 
+tsutils@^1.4.0:
```

Comment:
multiple tsutils again :( 

I've noticed yarn do this often lately when I try to upgrade a package (seems like a bug). I've usually fixed it by deleting all of the entries for a particular package (`tsutils` here) and running `yarn` again; it then resolves them to a single version.